<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TOMB MANAGEMENT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>711</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(photo/bg4.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>660</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Granite&quot;;
color: rgb(255, 191, 0);</string>
    </property>
    <property name="text">
     <string>YOUR  DATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 14pt &quot;Granite&quot;;</string>
    </property>
    <property name="text">
     <string>CHOOSE  YOUR  CHENG  MENG  DAY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>710</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Granite&quot;;
color: rgb(255, 191, 0);</string>
    </property>
    <property name="text">
     <string>MOST SELECTED  DATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 14pt &quot;Granite&quot;;</string>
    </property>
    <property name="text">
     <string>T O M B   I N F O R M A T I O N</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>851</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(photo/redbg.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Granite&quot;;
color: rgb(255, 191, 0);</string>
    </property>
    <property name="text">
     <string>TOMB  NUMBER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="showtombnumber">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 24pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="showservice">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Granite&quot;;
color: rgb(255, 191, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="getdate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>660</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
background-color: rgb(79, 0, 0);
font: 25 12pt &quot;Comfortaa Light&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>800</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 11pt &quot;Granite&quot;;
background-color: rgb(79, 0, 0);</string>
    </property>
    <property name="text">
     <string>UPDATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>800</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 11pt &quot;Granite&quot;;
background-color: rgb(79, 0, 0);</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QLabel" name="popmessage">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Granite&quot;;
color: rgb(255, 191, 0);</string>
    </property>
    <property name="text">
     <string>SERVICE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Granite&quot;;
color: rgb(255, 191, 0);</string>
    </property>
    <property name="text">
     <string>NAME ON TOMB</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="shownameontomb">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Granite&quot;;
color: rgb(255, 191, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(photo/tombconfirm.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="showdate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>710</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 24pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Granite&quot;;
color: rgb(255, 191, 0);</string>
    </property>
    <property name="text">
     <string> IN VITATION  CODE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getinvitationcode">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
background-color: rgb(79, 0, 0);
font: 25 12pt &quot;Comfortaa Light&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="showinvitationcode">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 12pt &quot;Comfortaa Light&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="popmessage2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>630</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sendcode">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>270</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(photo/bg4.jpg);
color: rgb(255, 191, 0);
font: 9pt &quot;Comfortaa&quot;;</string>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>label_2</zorder>
   <zorder>label_11</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>showtombnumber</zorder>
   <zorder>showservice</zorder>
   <zorder>getdate</zorder>
   <zorder>confirm</zorder>
   <zorder>back</zorder>
   <zorder>popmessage</zorder>
   <zorder>label_10</zorder>
   <zorder>shownameontomb</zorder>
   <zorder>label</zorder>
   <zorder>showdate</zorder>
   <zorder>label_12</zorder>
   <zorder>getinvitationcode</zorder>
   <zorder>popmessage2</zorder>
   <zorder>showinvitationcode</zorder>
   <zorder>sendcode</zorder>
  </widget>
 </widget>
 <resources>
  <include location="photo/convert.qrc"/>
 </resources>
 <connections/>
</ui>
